stable/dask/templates/NOTES.txt

```diff
@@ -1,3 +1,7 @@
+##########################################################
+# This chart has been deprecated in favor of stable/dask #
+##########################################################
```

Comment:
This is the wrong chart.

Reply:
Whoops!  My apologies for missing this.  Resolved.